apps/frontend/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Button, Input, InputGroup, InputRightElement } from '@chakra-ui/react';
+import { SmallCloseIcon } from '@chakra-ui/icons';
+
+type SearchBarProps = {
+
+    /* This function will filter data to be displayed in parent component.
+    Ex. Displaying names containing the characters typed in by the user.  */
+    onSearch: (search: string) => void;
```

Comment:
A comment giving example usage is probably better just put in the PR rather than in the code itself.